### DISCUSSION TITLE
Improve performance when used enabled composite files and minify css, js

### DISCUSSION THREAD

**Discussion by Mostafa-Moafi** [dnnsoftware/Dnn.Platform#5590] [Ideas] [answered]
Hi
Is it possible to generate a css and js file after bundling and minifying, for example, site.min.css and site.min.js?
if the answer is yes so we can use the preload meta tag for example :
```
<link rel="preload" href="site.min.css" />
<link rel="preload" href="site.min.js" />
```
This item can help us to improve it.
For more information :

[https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel/preload](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel/preload)

**Comment by valadas** [comment, marked as answer]
If bundling, it will create a single bundle with everything requested, so it won't be site.min.css. I will use an HTTP handler to request all the js and all the CSS in one go.

Also, maybe (including me) believe that bundling is a thing of the past as:
1. It does not support modules
2. If over HTTPS and HTTP2, you get parallel and lazy downloading which is more efficient than a single huge bundle.
